### PR TITLE
PAPR: check 3 masters install and minor update

### PR DESCRIPTION
**Pull request by vrutkovs**
Add another PAPR tests:
* [ ] install HA masters
```
TASK [ansible_service_broker : oc_secret] **************************************
Thursday 12 April 2018  13:53:03 +0000 (0:00:02.952)       0:45:04.570 ******** 
skipping: [ocp-master1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [ansible_service_broker : Create the Broker resource in the catalog] ******
Thursday 12 April 2018  13:53:03 +0000 (0:00:00.029)       0:45:04.600 ******** 
fatal: [ocp-master1]: FAILED! => {"changed": false, "msg": {"cmd": "/usr/local/bin/oc create -f /tmp/brokerout-k3_dtizl -n default", "results": {}, "returncode": 1, "stderr": "Error from server (ServiceUnavailable): error when creating \"/tmp/brokerout-k3_dtizl\": the server is currently unable to handle the request (post clusterservicebrokers.servicecatalog.k8s.io)\n", "stdout": ""}}
```
Seems something is wrong with service broker, maybe worth disabling it for now?

* [ ] minor update

Fails on `Upgrade all storage`:

* some objects were not yet available in API?
* is there a need to migrate storage in 3.10?

@smarterclayton WDYT?

### CODE REVIEW

**Comment by vrutkovs** [comment]
2nd and 3rd master don't get aggregate crt:
```
aggregatorConfig.proxyClientInfo.certFile: Invalid value: "/etc/origin/master/aggregator-front-proxy.crt": could not read file: stat /etc/origin/master/aggregator-front-proxy.crt: no such file or directory
```

**Comment by smarterclayton** [comment]
Aggregation is fixed in my latest upgrade commit.

**Comment by vrutkovs** [comment]
/retest

Something is up with CI infra - `Failure talking to yum: Could not contact any CDS load balancers: rhui2-cds01.us-east-1.aws.ce.redhat.com, rhui2-cds02.us-east-1.aws.ce.redhat.com.`

**Comment by sdodson** [comment]
https://github.com/openshift/origin/issues/19295

**Comment by vrutkovs** [comment]
```TASK [Upgrade all storage] *****************************************************
Friday 13 April 2018  08:14:27 +0000 (0:00:00.041)       0:03:53.635 ********** 
fatal: [ocp-master]: FAILED! => {"changed": true, "cmd": ["/usr/local/bin/oc", "adm", "--config=/etc/origin/master/admin.kubeconfig", "migrate", "storage", "--include=*", "--confirm"], "delta": "0:00:04.076172", "end": "2018-04-13 08:14:17.810138", "failed_when_result": true, "msg": "non-zero return code", "rc": 1, "start": "2018-04-13 08:14:13.733966", "stderr": "error: the server doesn't have a resource type \"clusterservicebrokers\"", "stderr_lines": ["error: the server doesn't have a resource type \"clusterservicebrokers\""], "stdout": "", "stdout_lines": []}```

Probably its a flake

bot, retest this please

**Comment by vrutkovs** [comment]
bot, retest this please

/retest

**Comment by vrutkovs** [comment]
bot, retest this please

File: roles/etcd/defaults/main.yaml [hunk 1]
```diff
@@ -4,8 +4,10 @@ r_etcd_common_backup_sufix_name: ''
 
 l_is_etcd_system_container: "{{ (openshift_use_etcd_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
 
+l_etcd_static_pod: "{{ not (r_etcd_common_skip_command_shim is defined and r_etcd_common_skip_command_shim) or openshift.node.bootstrapped }}"
```
Comment (michaelgugino) [inline roles/etcd/defaults/main.yaml]: please tell me bootstrapped didn't make it's way into openshift_facts ?

File: roles/etcd/defaults/main.yaml [hunk 2]
```diff
@@ -17,7 +19,12 @@ etcd_image: "{{ etcd_image_dict[openshift_deployment_type | default('origin')] }
 # etcd run on a host => use etcdctl command directly
 # etcd run as a docker container => use docker exec
 # etcd run as a runc container => use runc exec
-r_etcd_common_etcdctl_command: "{{ 'etcdctl' if (r_etcd_common_etcd_runtime == 'host') else ('docker exec etcd_container etcdctl' if (r_etcd_common_etcd_runtime == 'docker') else 'runc exec etcd etcdctl') }}"
+etcdctl_dict:
```
Comment (michaelgugino) [inline roles/etcd/defaults/main.yaml]: While technically correct, I liked the previous implementation due to the fact that we were getting rid of half of it in 3.11.  This way is less clear, it's not clear what's doing what or why.

File: roles/etcd/defaults/main.yaml (hunk 1 above)
Comment (vrutkovs) [reply roles/etcd/defaults/main.yaml]: Its being set in playbooks/init/cluster_facts.yml:
```
    openshift_facts:
      role: node
      local_facts:
        sdn_mtu: "{{ openshift_node_sdn_mtu | default(None) }}"
        bootstrapped: "{{ openshift_is_bootstrapped }}"

```

File: roles/etcd/defaults/main.yaml (hunk 2 above)
Comment (vrutkovs) [reply roles/etcd/defaults/main.yaml]: Right, I'm not entirely sure which options should be removed - so I kept those. FAH tests would verify only minor update of static pods case

**Comment by vrutkovs** [comment]
Service catalog issue seems to be related to https://github.com/openshift/origin/issues/18119

**Pull request merged**